sites-enabled/10-www.freecodecamp.org.conf

```diff
@@ -116,7 +122,7 @@ server {
   # SSL
   include snippets/common/ssl-freecodecamp-org.conf;
 
-  return 301 https://www.freecodecamp.org$request_uri;
+  return 302 https://www.freecodecamp.org$request_uri;
```

Comment:
Good question. 

In this case, the server block is for the host `.freecodecamp.org`. This is a catch all for, and superseded when we add something on a subdomain. 

While we well almost never have SSL terminating outside of CF if we control the deployed on a subdomain. 

This helps us avoid the night mares of a 301 redirect during initial setup, were maybe testing stuff with a LE cert and not necessary having deployed our origin certs.

In short, 301s are bad when you are uncertain of what you may end up doing to a config, in this case a subdomain that we are not sure we may be needing in a certain way.

Reply:
Okay, I think I get that.  In which case, shouldn't we be using 301 for this and 302 for chinese?  Maybe I'm still missing something, since this `server_name` is specifically `www.freecodecamp.org` not `*.freecodecamp.org` (for that I could see us wanting a 302).

Reply:
Yeah if you visit the file for the last commit, it should be clear. BTW, a `.freecodecamp.org`, notice the dot, is a catch all.

Reply:
Thanks. I'm not sure how I missed the `.` I need more sleep/coffee.